monin_obukhov/include/monin_obukhov.inc

```diff
@@ -123,6 +123,11 @@ do j=1, n3
   enddo
 enddo
 
+do j=1, n3
```

Comment:
ah it did that for you  too!  the do loops are repeated here

Reply:
i think something wonky might be happening with our branches, might be better to simplify things and just use mixedmode as the starting point going forward